common/lib/capa/capa/responsetypes.py

```diff
@@ -831,16 +841,110 @@ def assign_choice_names(self):
             if not choice.get('id'):
                 choice.set("id", chr(ord("A") + index))
 
+    def grade_via_edc(self, all_choices, student_answer, student_non_answers):
+        """
+        Calculates partial credit on the EDC scheme.
+        For each correctly selected or correctly blank choice, score 1 point.
+        Divide by total number of choices.
+        Returns a CorrectMap.
```

Comment:
I was definitely not proposing changing variable names. The difference here is on internal interface vs. external. In this case, the former happens to be variables and the latter, the function name. When I code, I try to be verbose on public interfaces, and concise on internals. 

External code becomes much more self-documenting then. If I have a chunk of code elsewhere which calls `bar.bif.grade_via_every_decision_counts()` I know what it's talking about without additional docs or digging deeper. Inside of the function, it's both obvious and unnecessarily verbose to not use the acronym. 

Again, that's just my style. If it's just in the docstring, I'd be happy with it as well.